functests/sctp/sctp.go

```diff
@@ -75,21 +81,17 @@ var _ = Describe("sctp", func() {
 					LabelSelector: "node-role.kubernetes.io/worker,!" + strings.Replace(sctpNodeSelector, "=", "", -1),
 				})
 				Expect(err).ToNot(HaveOccurred())
-
 				Expect(len(nodes.Items)).To(BeNumerically(">", 0))
-				clientNode = nodes.Items[0].ObjectMeta.Labels[hostnameLabel]
 				serverNode = nodes.Items[0].ObjectMeta.Labels[hostnameLabel]
-				if len(nodes.Items) > 1 {
-					serverNode = nodes.Items[1].ObjectMeta.Labels[hostnameLabel]
-				}
+
 				createSctpService(client.Client)
 			})
 			Context("Client Server Connection", func() {
 				// OCP-26995
-				It("Should NOT connect a client pod to a server pod", func() {
+				It("Should NOT start a server pod", func() {
 					By("Starting the server")
 					serverArgs := []string{"-ip", "0.0.0.0", "-port", "30101", "-server"}
-					pod := scptTestPod("scptserver", serverNode, "sctpserver", serverArgs)
+					pod := scptTestPod("scptserver", serverNode, "sctpserver", testNamespace, serverArgs)
```

Comment:
scpt -> sctp

Reply:
Also I think you can use this function: `startServerPod`

and pass a pod status to it so we don't have a duplicate code

Reply:
Uhm, I am not completely happy about that. Start should start (and expect to have the server started).
Having a tiny duplication felt better than having that function serve too many purposes.